client/state/data-layer/wpcom/read/streams/index.js

```diff
@@ -301,6 +358,9 @@ export function handlePage( action, data ) {
 	} else {
 		actions.push( receivePosts( streamPosts ) );
 		actions.push( receivePage( { streamKey, query, streamItems, pageHandle, gap } ) );
+		if ( streamSites.length > 0 ) {
+			actions.push( receiveRecommendedSites( { seed: 'seed?', sites: streamSites } ) );
```

Comment:
I noticed `recieveRecommendedSites` was filtering out some of the results of `streamSites` when i tried it earlier (ones that i wouldn't initially expect to be filtered out here). This could be something to circle back to and investigate to see if we need to create a new action/selector combo for something like `recieveDiscoverSites` or not.